mlir-tensorrt/tensorrt/lib/TensorRT/Transforms/RaiseActivations.pdll

```diff
@@ -68,17 +108,107 @@ Constraint Add(lhs: Value, rhs: Value) -> Op {
 Constraint Tanh(x: Value) -> Op {
   return TanhConstraint(op<tensorrt.activation>(x));
 }
+Constraint Max(lhs: Value, rhs: Value) -> Op {
+  return MaxConstraint(op<tensorrt.element_wise>(lhs, rhs));
+}
+Constraint Min(lhs: Value, rhs: Value) -> Op {
+  return MinConstraint(op<tensorrt.element_wise>(lhs, rhs));
+}
+Constraint Pow(lhs: Value, rhs: Value) -> Op {
+  return PowConstraint(op<tensorrt.element_wise>(lhs, rhs));
+}
+Constraint Erf(x: Value) -> Op {
+  return ErfConstraint(op<tensorrt.unary>(x));
+}
+
+Rewrite GetSplatElementAttr(x: Value) -> Attr [{
+  while(true) {
+    if(auto expandRank = x.getDefiningOp<tensorrt::ExpandRankOp>())
+      x = expandRank.getInput();
+    else if(auto reshape = x.getDefiningOp<tensorrt::ReshapeOp>())
+      x = reshape.getInput();
+    else if(auto broadcast = x.getDefiningOp<tensorrt::BroadcastOp>())
+      x = broadcast.getInput();
+    else if(auto cast = x.getDefiningOp<tensorrt::CastOp>())
+      x = cast.getInput();
+    else if(auto identity = x.getDefiningOp<tensorrt::IdentityOp>())
+      x = identity.getInput();
+    else if(auto slice = x.getDefiningOp<tensorrt::SliceOp>())
+      x = slice.getInput();
+    else if(auto constant = x.getDefiningOp<tensorrt::ConstantOp>()) {
+      DenseElementsAttr els{};
+      if(!matchPattern(x, m_Constant(&els)))
+        return {};
+      if(!els.isSplat())
+        return {};
+      Attribute value = els.getSplatValue<Attribute>();
+      return value;
+    } else
+      return {};
+  }
+  return {};
+}];
+
+Constraint HasSplatElements(x: Value) [{
+  while(true) {
+    if(auto expandRank = x.getDefiningOp<tensorrt::ExpandRankOp>())
+      x = expandRank.getInput();
+    else if(auto reshape = x.getDefiningOp<tensorrt::ReshapeOp>())
+      x = reshape.getInput();
+    else if(auto broadcast = x.getDefiningOp<tensorrt::BroadcastOp>())
```

Comment:
Add a helper function to RaiseActivations.cpp to eliminate redundant code

Reply:
I have added a helper function to `Utils.cpp`.  This is the same helper function as on PR #680.